src/garage/torch/algos/vpg.py

```diff
@@ -128,92 +134,133 @@ def train_once(self, itr, paths):
             paths (list[dict]): A list of collected paths
 
         Returns:
-            dict: Processed sample data, with key
-                * average_return: (float)
+            numpy.float64: Calculated mean value of undiscounted returns
 
         """
         obs, actions, rewards, valids, baselines = self.process_samples(
             itr, paths)
 
-        loss = self._compute_loss(itr, obs, actions, rewards, valids,
-                                  baselines)
-
-        self._old_policy.load_state_dict(self.policy.state_dict())
+        if self._maximum_entropy:
+            policy_entropies = self._compute_policy_entropy(obs)
+            rewards += self._policy_ent_coeff * policy_entropies
```

Comment:
entropy penalty is moved outside of `compute_loss()`, how can TRPO enable `_maximum_entropy`, then?

Reply:
what do you mean by "enable `_maximum_entropy`"?
TRPO can enable `_maximum_entropy` by setting `entropy_method` = 'max'

also TRPO and VPG share the `train_once` method. (which contains the code)

Reply:
@naeioi are you satisfied?